x/ecocredit/module/module.go

```diff
@@ -79,11 +79,23 @@ func (a Module) RegisterInterfaces(registry types.InterfaceRegistry) {
 
 func (a Module) RegisterServices(configurator servermodule.Configurator) {
 	server.RegisterServices(configurator, a.paramSpace, a.accountKeeper, a.bankKeeper, a.distributionKeeper)
+
+	err := configurator.RegisterMigration(ecocredit.ModuleName, 2,
```

Comment:
shouldn't that be 1 since we're migrating from consensus version 1 to 2?
```suggestion
	err := configurator.RegisterMigration(ecocredit.ModuleName, 1,
```

Reply:
Correct, `forVersion` was confusing.